### ISSUE TITLE
Invoking Call with an invalid method will silently fail in C#

### ISSUE THREAD

**Issue by andy-noisyduck** [godotengine/godot#46645]
**Godot version:**
3.2.3 (Mono)

**OS/device including version:**
Win 10

**Issue description:**
Any invocation of `Call("method")` to a method that does not exist will silently fail in C#. It will continue on as if the invocation was successful. This is obviously incorrect behaviour. The GDscript equivalent of the same code will correctly raise a "Nonexistent function" error.

**Steps to reproduce:**
``` 
// This will be silently ignored
Call("MethodThatDoesNotExist");
```

The equivalent GDScript code `call("method_that_does_not_exist");` will correctly give the error `Invalid call. Nonexistent function 'method_that_does_not_exist (via call)'

**Minimal reproduction project:**
[Testing-Call.zip](https://github.com/godotengine/godot/files/6081134/Testing-Call.zip)
Repro project includes both C# and GDS versions of script.

**Comment by andy-noisyduck** [comment]
Not sure if https://github.com/godotengine/godot/issues/43282 (which seems to be specifically for `Tool` scripts) is also due to this bug.

**Comment by raulsntos** [comment]
- Sounds like a duplicate of https://github.com/godotengine/godot/issues/59830.
- Should be fixed by https://github.com/godotengine/godot/pull/79249 for 4.x. I understand you are unable to move to 4.x so I'll be keeping this issue open for 3.x.

**Comment by akien-mga** [comment]
Fixed by #79433.
